ZAPD/ZAudio.h

```diff
@@ -92,37 +92,37 @@ class ZAudio : public ZResource
 	std::vector<AudioTableEntry> sequenceTable;
 	std::vector<AudioTableEntry> sampleBankTable;
 	std::vector<std::vector<char>> sequences;
-	std::map<uint32_t, SampleEntry*> samples;
+	std::map<uint64_t, SampleEntry*> samples;
```

Comment:
What's the reasoning for using `uint64_t`?
If it is for 64bit systems, should this instead be `uintptr_t`?

Reply:
It's an artifact of some things I was trying that were removed. I plan on changing it back.